lib/wanda/operations/catalog/registry.ex

```diff
@@ -18,7 +20,9 @@ defmodule Wanda.Operations.Catalog.Registry do
     "sapinstancestart@v1" => SapInstanceStartV1.operation(),
     "sapinstancestop@v1" => SapInstanceStopV1.operation(),
     "saptuneapplysolution@v1" => SaptuneApplySolutionV1.operation(),
-    "saptunechangesolution@v1" => SaptuneChangeSolutionV1.operation()
+    "saptunechangesolution@v1" => SaptuneChangeSolutionV1.operation(),
+    "enablepacemaker@v1" => EnablePacemakerV1.operation(),
```

Comment:
Let's try to keep and alphabethical order on the map.
It helps to me at least to find things when it gets big